plugins/outputs/warp10/warp10_test.go

```diff
@@ -0,0 +1,22 @@
+package warp10
```

Comment:
We need to add some more substantial tests, particularly around the creation of the request body.